pages/common/jtbl.md

```diff
@@ -0,0 +1,20 @@
+# jtbl
+
+> Utility to print JSON and JSON Lines data as a table in the terminal.
+> More information: <https://github.com/kellyjonbrazil/jtbl>.
+
+- Print a table from JSON or JSON Lines input:
+
+`cat {{file.json}} | jtbl`
+
+- Specify the column width for wrapping:
+
+`cat {{file.json}} | jtbl --cols={{width}}`
+
+- Truncate rows instead of wrapping:
+
+`cat {{file.json}} | jtbl -t`
+
+- Don't wrap or truncate rows:
+
+`cat {{file.json}} | jtbl -n`
```

Comment:
I like when descriptions tell you exactly what they do, rather than just making the user infer.

So for example command 2 `Print a table from JSON or JSON Lines input and truncate rows instead of wrapping them:

Reply:
Sure.